### ISSUE TITLE
Include SVG styling for SVG Files

### ISSUE THREAD

**Issue by BrentM** [mindok/contex#24]
It would be handy to generate a SVG with the XML styling that could be served as a img.

I put this in a Phoenix controller:

```
@svg ["<?xml version=\"1.0\" encoding=\"utf-8\"?><svg version=\"1.1\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">"]
  
  def plot(conn, params) do
    data = [
        {0.0, 0.0, "Hippo"},
        {0.5, 0.3, "Turtle"},
        {0.4, 0.3, "Turtle"},
        {0.2, 0.3, "Rabbit"}
        ]
    dataset = Contex.Dataset.new(data, ["x", "y", "category"])
    plot_content = Contex.BarChart.new(dataset)
    plot = Contex.Plot.new(600, 400, plot_content)
    {:safe, output} = Contex.Plot.to_svg(plot)

    conn = put_resp_content_type(conn, "image/svg+xml")
    send_resp(conn, 200, @svg ++ output ++ ["</svg>"])
```

I just surrounded the SVG from to_plot with this: `@svg ++ output ++ ["</svg>"]`
It works but isn't graceful.

I should be able to include SVGs in img tags instead of including them in templates.

Would something like this be a reasonable addition?:
```
def to_svg_file(plot) do
  {:safe, svg} = to_svg(plot)
  svgfile = @svgxml ++ svg ++ ["</svg>"]
  svgfile
end
```
This could also be useful for generating static svgs.

**Comment by mindok** [comment]
Thanks @BrentM - makes sense. It should really be properly marked up with XML namespace etc anyway now that you mention it! I'll leave this open and pick it up in the next campaign of enhancements, or feel free to propose a PR.

**Comment by mindok** [comment]
Fixed by PR #25
